src/main/java/org/minions/devfund/sergio/battleship_game/Ocean.java

```diff
@@ -0,0 +1,53 @@
+package org.minions.devfund.sergio.battleship_game;
+
+import java.util.Random;
+
+public class Ocean {
+
+    private int shots = 0;
+    private int hitCount = 0;
+    private int sunkShips = 0;
+
+    public Ocean() {
+
+    }
+
+    public void placeAllShipsRandomly() {
+
+        Random random = new Random();
+        Ship[] ships = new Ship[13];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "ships". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "ships" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)